DragaliaAPI/Features/Event/EventDropService.cs

```diff
@@ -165,6 +165,19 @@ QuestData quest
         return dropList;
     }
 
+    private bool IsEligibleForT3Drops(QuestData quest, EventData evt)
+{
+    // Check for specific event names
+    if (evt.EventName == "Accursed Archives" || evt.EventName == "Dream Big Under the Big Top")
```

Comment:
This check will not work. the [EventData class](https://github.com/sapiensanatis/Dawnshard/blob/44a089c573441f1f64dbd09d1595195ca8d30dfc/DragaliaAPI.Shared/MasterAsset/Models/Event/EventData.cs) doesn't have an EventName property, you have to check the corresponding event using the actual event id

Reply:
The two event ids should be 20831 (The Accursed Archives) and 20820 (Dream under the Big Top)

Reply:
Hi @LukeFZ , I've updated it like you mentioned. Can you please check now?